tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -390,3 +403,37 @@ Verify Anaconda Success Message Based On Version
     ELSE
         Success Message Should Contain    ${ANACONDA_DISPLAYED_NAME_LATEST}
     END
+
+Verify Operator Is Added On ODS Dashboard
+    [Documentation]     It checks operator is present on ODS Dashboard in Enabled section
+    [Arguments]         ${operator_name}
+    Launch Dashboard   ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    ...   ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Wait Until Keyword Succeeds    10x    1m    Verify Service Is Enabled  app_name=${operator_name}
+    Close Browser
+
+Uninstall Operator And Check Enabled Page Is Rendering
+    [Documentation]    Uninstall Operator And Check Enabled Page(ODS) Is Rendering, Not shwoing "Error loading components"
+    [Arguments]     ${operator_name}    ${operator_appname}
+    Open Installed Operators Page
+    Uninstall Operator    ${operator_name}
+    Launch Dashboard   ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    ...   ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Page Should Not Contain    Error loading components
+    Close All Browsers
```

Comment:
maybe this is not really needed, because the Teardown already closes all the browsers

Reply:
done